COPYRIGHT.txt

```diff
@@ -146,17 +146,16 @@ Comment: DroidSans font
 Copyright: 2008, The Android Open Source Project
 License: Apache-2.0
 
+Files: ./thirdparty/fonts/JetBrainsMono_Regular.ttf
+Comment: JetBrains Mono font
+Copyright: 2020, JetBrains s.r.o.
+License: OFL-1.1
+
 Files: ./thirdparty/fonts/NotoSans*.ttf
 Comment: Noto Sans font
 Copyright: 2012, Google Inc.
 License: OFL-1.1
 
-Files: ./thirdparty/fonts/Hack_Regular.ttf
```

Comment:
Is there a way that we can include both in the engine? One would be the default, the other would be selectable in the editor settings menu without having to download a .ttf or .otf file and pointing Godot to it.

Reply:
They're too similar, I don't see much point to include two similar fonts for the same purpose. Most users will use the defaults, though who want something non-default will use their own font.